src/main/scala/com/wavesplatform/UtxPool.scala

```diff
@@ -38,15 +38,15 @@ class UtxPool(
   private def removeExpired(currentTs: Long): Unit =
     transactions.entrySet().removeIf(tx => (currentTs - tx.getValue.timestamp).millis > utxSettings.maxTransactionAge)
 
-  def putIfNew(tx: Transaction, source: Option[Channel] = None): Either[ValidationError, Transaction] =
+  def putIfNew(tx: Transaction, source: Option[Channel] = None, broadcast: Boolean = true): Either[ValidationError, Transaction] =
```

Comment:
`source` parameter doesn't make any sense when `broadcast` is false
Not only that, but the whole `putIfNew` signature now takes responsibility and details on what should happen after.
I suugest we remove `allChannels` from `UtxPool` and just the broadcasting later